Makefile

```diff
@@ -33,6 +32,7 @@ publish:
 	@make clean
 	@printf "\nPublishing lib"
 	@make setup
+	@poetry config pypi-token.pypi $(PYPI_API_TOKEN)
```

Comment:
Why do you need to move this to here if there is a call to the `setup` into the previous line? Isn't that enough to get it done?
Also, we can use "setup" as a [dependency](https://web.mit.edu/gnu/doc/html/make_2.html) instead of calling it using `@make`

Reply:
This line must be executed on publish only